lib/TwoFactorAuth.php

```diff
@@ -54,7 +54,7 @@ public function __construct(
     public function createSecret(int $bits = 80, bool $requirecryptosecure = true): string
     {
         $secret = '';
-        $bytes = (int) ceil($bits / 5);   // We use 5 bits of each byte (since we have a 32-character 'alphabet' / BASE32)
+        $bytes = (int)ceil($bits / 5);   // We use 5 bits of each byte (since we have a 32-character 'alphabet' / BASE32)
```

Comment:
I find a space after the type coercion much more readable

Reply:
This is also PHP-CS-Fixer's doing. It's in the rules to format it like such.